addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24712,3 +24712,14 @@ msgid "50th Anniversary Edition"
 msgstr ""
 
 # 40000 to 40800 are reserved for Video Versions feature
+
+#: system/settings/settings.xml
```

Comment:
Please add `#.` line, like you did for the other string you've introduced, explaining in clear words what this setting is actually controlling. Our translators need this.

Reply:
Done

Reply:
Well, is it helpful just to repeat the string? Not really. Please look how others did it, explaining where and for what exactly it is used, like "label for a setting to configure whether video files ..."